site/templates/siteComponent.pug

```diff
@@ -180,7 +180,7 @@ html(lang='en-US').spectrum.spectrum--light.spectrum--medium
             if examples.length
               header.spectrum-CSSComponent-sectionHeading(id="variants")
                 h4.spectrum-Heading3
-                  a(href="#variants").spectrum-BigSubtleLink Variants
+                  a(href="#variants").spectrum-BigSubtleLink=`${component.examplesHeading ? component.examplesHeading : 'Variants'}`
```

Comment:
This just makes it possible for the Typography landing page to say "Components" instead of "Variants"